oereb_einzelschutz_denkmal/insert_oereb_einzelschutz_denkmal_tables.sql

```diff
@@ -233,8 +233,8 @@ localiseduri AS
         'de' AS alanguage,
         CASE
             WHEN rechtsvorschrften_dokument.multimedia_link IS NULL
-                THEN 'https://geo.so.ch/docs/ch.so.ada.denkmal/pdf_404.pdf'
-            ELSE regexp_replace(rechtsvorschrften_dokument.multimedia_link, 'artplus\.verw\.rootso\.org/MpWeb-apSolothurnDenkmal/download/(.*)\?mode=gis','geo.so.ch/docs/ch.so.ada.denkmal/ada_\1.pdf')
+                THEN 'https://'||${s3AdaLiveBucket}||'/pdf_404.pdf'
+            ELSE regexp_replace(rechtsvorschrften_dokument.multimedia_link, 'artplus\.verw\.rootso\.org/MpWeb-apSolothurnDenkmal/download/(.*)\?mode=gis', ${s3AdaLiveBucket}||'/ada_\1.pdf')
```

Comment:
Das wäre dann auch hier so, ungefähr:
```suggestion
            ELSE regexp_replace(rechtsvorschrften_dokument.multimedia_link, 'artplus\.verw\.rootso\.org/MpWeb-apSolothurnDenkmal/download/(.*)\?mode=gis', 's3.eu-central-1.amazonaws.com/'||${s3AdaLiveBucket}||'/ada_\1.pdf')
```